frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -500,6 +478,7 @@ class DashboardHeaderContainer extends Component<DashboardHeaderProps> {
         />,
       );
 
+      // I don't think this is used anymore
```

Comment:
yeah, i think it's only accessible from the "last edit by ..." thing or the "info" sidebar. Let's remove it